### PR TITLE
Update sidebar items to match MVP design

### PR DESCRIPTION
**Pull request by dmkirshon**
- Update SideBar Navigation references and style to match to Figma designs for MVP release

Includes:
 - Addition of extra sidebar section for Support and external navigation link component
 - Removal of logout from sidebar
 - Addition of a Settings Page navigation link (Settings Page and Terms of Service linke created via separate task)
 
| Previous | New |
|--------|--------|
| CU-85zrqemzr_SideBar-Items-Previous | CU-85zrqemzr_SideBar-Items-Updated |

### CODE REVIEW

File: src/components/home/SideBarExternalNavLink.tsx [hunk 1]
```diff
@@ -0,0 +1,60 @@
+import { FunctionComponent } from "react";
+import { Stack } from "@mui/material";
+import theme from "theme";
+
+const activeBackground = "rgba(255, 255, 255, 0.1)";
```
Comment (escobarjonatan) [inline src/components/home/SideBarExternalNavLink.tsx:5]: might be good to put this in the theme file so we can reuse it in other places 🤔

File: src/components/home/SideBarExternalNavLink.tsx [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+        data-testid="navStyled"
+      >
+        { icon }
+        <span>{ label }</span>
+      </Stack>
+    </a>
+  );
+};
+
+export default SideBarExternalNavLink;
```
[the remark this answers is not in the document]
Comment (dmkirshon) [reply src/components/home/SideBarExternalNavLink.tsx:60]: 👌 refactoring and removing the `SideBarExternalNavLink` component

File: src/components/home/SideBarExternalNavLink.tsx (hunk 1 above)
Comment (dmkirshon) [reply src/components/home/SideBarExternalNavLink.tsx:5]: 👌 Adding to the theme.colors section

The nav link might need a typography specific section as well if we do reuse, or want a majority of styling to live in one place, but for now this seems like a unique item for the sidebar

**Comment by dmkirshon** [comment]
> LGTM. I assume the page at https://newm.io/faq_artist-portal just hasn't been built yet? Also the Telegram link just opened the Telegram home page for me, but I assume that's because I haven't created a Telegram account yet.

Yes, the newm faq will be something for marketing/product to put together before MVP release.

Double checking with marketing, the link provided was the specific Telegram url for the NEWM artist support, the group might not have been created yet because other telegram links do have a different join group page and don't go straight to the home Telegram site.

**Review by escobarjonatan — APPROVED** [review]

**Comment by dmkirshon** [comment]
> > LGTM. I assume the page at https://newm.io/faq_artist-portal just hasn't been built yet? Also the Telegram link just opened the Telegram home page for me, but I assume that's because I haven't created a Telegram account yet.
> 
> Yes @scandycuz, the newm faq will be something for marketing/product to put together before MVP release.
> 
> Double checking with marketing, the link provided was the specific Telegram url for the NEWM artist support, the group might not have been created yet because other telegram links do have a different join group page and don't go straight to the home Telegram site.

From marketing discussion: updated the newm.io faq and artist telegram group links to match with the planned MVP rollout

Support should be correct but might be updated Pre-MVP

**Pull request merged**